novaagent/libs/centos.py

```diff
@@ -159,6 +161,10 @@ def _setup_routes(self, ifname, iface):
     def version_tuple():
         """Tuple representation of os version"""
         version = distro.version().split('.')
+        if len(version) < 2:
+            # Tuple comparision doesn't like comparing different sized tuples.
+            version.append(0)
+
```

Comment:
Comparing different sized tuples works fine in Python.

Reply:
I will roll this change back, to bring out the error I was seeing.